main.py

```diff
@@ -0,0 +1,58 @@
+import random
+
+
+def get_list_of_users_names() -> list:
+    """
+    The function asks the user to input a list of names, checks and returns it.
+    """
+    users_input = input("Input list of names separated by comma: ").split(",")
+    users_list_of_names = [name.strip() for name in users_input]
+
+    # checking a number of entered names
+    while len(users_list_of_names) < 2:
+        users_input = input("Input more than one name, please: ").split(",")
+        users_list_of_names = [name.strip() for name in users_input]
+
```

Comment:
May be better to turn into a set rather than a list at this point to remove any duplicate names entered by the user.